### PR TITLE
Validation of MAUI Preview

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
This supports iOS, Android, and UWP. It supports only Basic and Broker projects. 
Intune is not supported for any platform.

### CODE REVIEW

File: 2-With-broker/UserDetailsClient/UserDetailsClient.UWP/UserDetailsClient.UWP.csproj [hunk 1]
```diff
@@ -128,7 +128,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Client" Version="4.45.0" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="[4.46.0-preview2]" />
```
Comment (bgavrilMS) [inline 2-With-broker/UserDetailsClient/UserDetailsClient.UWP/UserDetailsClient.UWP.csproj:131]: did you put in the version manually or did nuget do it? Interesting use of floating versions.

**Comment by bgavrilMS** [comment]
A lot of customers push back against preview versions. So I'm thinking that our official sample should not use the -preview. It is fine to put this in a branch I think.

**Comment by bgavrilMS** [comment]
You are not upgrading this sample to use Maui, so the Intune project can use the last MSAL.

**Review by bgavrilMS — APPROVED** [review]

File: 2-With-broker/UserDetailsClient/UserDetailsClient.UWP/UserDetailsClient.UWP.csproj (hunk 1 above)
Comment (SameerK-MSFT) [reply 2-With-broker/UserDetailsClient/UserDetailsClient.UWP/UserDetailsClient.UWP.csproj:131]: It was done by NuGet.

**Pull request closed without merging**